functional_tests/functional_test.go

```diff
@@ -234,6 +236,19 @@ func deployChartsAndApps(t *testing.T) {
 			require.NoError(t, err)
 		}
 	}
+	// .NET test app
+	stream, err = os.ReadFile(filepath.Join(testDir, "dotnet", "deployment.yaml"))
+	require.NoError(t, err)
+	deployment, _, err = decode(stream, nil, nil)
+	require.NoError(t, err)
+	_, err = deployments.Create(context.Background(), deployment.(*appsv1.Deployment), metav1.CreateOptions{})
+	if err != nil {
+		_, err2 := deployments.Update(context.Background(), deployment.(*appsv1.Deployment), metav1.UpdateOptions{})
```

Comment:
So this test code duplicated. I do appreciate the input. I'm going to consider this test code out of scope for now because it is duplicate test code and would be an improvement. In the future, we can improve this test code.